website/pages/docs/internals/telemetry.mdx

```diff
@@ -0,0 +1,346 @@
+| `vault.expire.fetch-lease-times`          | Time taken to fetch lease times                                             | ms     | summary |
+| `vault.expire.fetch-lease-times-by-token` | Time taken to fetch lease times by token                                    | ms     | summary |
+| `vault.expire.num_leases`                 | Number of all leases which are eligible for eventual expiry                 | leases | gauge   |
+| `vault.expire.revoke`                     | Time taken to revoke a token                                                | ms     | summary |
+| `vault.expire.revoke-force`               | Time taken to forcibly revoke a token                                       | ms     | summary |
+| `vault.expire.revoke-prefix`              | Time taken to revoke tokens on a prefix                                     | ms     | summary |
+| `vault.expire.revoke-by-token`            | Time taken to revoke all secrets issued with a given token                  | ms     | summary |
+| `vault.expire.renew`                      | Time taken to renew a lease                                                 | ms     | summary |
+| `vault.expire.renew-token`                | Time taken to renew a token which does not need to invoke a logical backend | ms     | summary |
+| `vault.expire.register`                   | Time taken for register operations                                          | ms     | summary |
+
+These operations take a request and response with an associated lease and register a lease entry with lease ID
+
+| Metric                       | Description                                                                                   | Unit | Type    |
+| :--------------------------- | :-------------------------------------------------------------------------------------------- | :--- | :------ |
+| `vault.expire.register-auth` | Time taken for register authentication operations which create lease entries without lease ID | ms   | summary |
+| `vault.policy.get_policy`    | Time taken to get a policy                                                                    | ms   | summary |
+| `vault.policy.list_policies` | Time taken to list policies                                                                   | ms   | summary |
+| `vault.policy.delete_policy` | Time taken to delete a policy                                                                 | ms   | summary |
+| `vault.policy.set_policy`    | Time taken to set a policy                                                                    | ms   | summary |
+| `vault.token.create`         | The time taken to create a token                                                              | ms   | summary |
+| `vault.token.create_root`    | Number of created root tokens. Does not decrease on revocation.                               |
+| token                        | counter                                                                                       |
```

Comment:
This shouldn't be on a new line, it's part of L138

Reply:
Great catch thank you, just pushed a fix